scripts/play.ts

```diff
@@ -0,0 +1,380 @@
+import mri from "mri";
+import chalk from "chalk";
+import * as fs from "node:fs/promises";
+import * as fsSync from "node:fs";
+import * as path from "node:path";
+import * as crypto from "node:crypto";
+
+import { exec } from "./utils/commandUtils";
+
+const PLAYGROUNDS_ROOT = path.join(process.cwd(), "playgrounds");
+const DEFAULT_FRAMEWORK = "next" satisfies Args["framework"];
+
+run();
+
+type Args = {
+  "dry-run": boolean;
+  start: boolean;
+  new: boolean;
+  framework?: "next" | "nuxt" | "sveltekit" | (string & {});
+};
+
+type PackageJSON = {
+  name: string;
+  scripts: Record<string, string>;
+};
+
+type SliceMachineConfig = {
+  localSliceSimulatorURL?: string;
+  apiEndpoint?: string;
+};
+
+/**
+ * The root command.
+ */
+async function run(): Promise<void> {
```

Comment:
I figured `run` make sense here since this is a script. `main` could be another option.

Names like `run` and `main` differentiate itself from the script's domain logic, like `createPlayground`. `play` could be confused with something underlying logic.

To me, `run` (or `main`) a clear way to declare "this is the top-level function that runs the script."

I don't feel too strongly about it, so if you think `play` is still a better option, I will change it. 🙂

Reply:
Let's go with `main` then. 🙂